lib/WebService/MinFraud/Record/CreditCard.pm

```diff
@@ -35,6 +42,12 @@ has is_prepaid => (
     coerce  => BoolCoercion,
 );
 
+has type => (
+    is        => 'ro',
+    isa       => Maybe [ Enum [qw( charge credit debit )] ],
```

Comment:
Not sure about this validation. Do we want this to fail if we decide to add a new type?